<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkWindow</class>
 <widget class="QMainWindow" name="WorkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="startPoint">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>170</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>140</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <property name="text">
     <string>## 出发点:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label1_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">textBrowser-&gt;setAttribute(Qt::WA_TranslucentBackground);</string>
    </property>
    <property name="text">
     <string>## 终点:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="endPoint">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>270</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startConfirm">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>170</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endConfirm">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>270</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startSearch">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>470</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始查找</string>
    </property>
   </widget>
   <widget class="QLabel" name="startPointWarn">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>220</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="endPointWarn">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>320</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="SubwayMap" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>771</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="administrator">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>530</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>管理系统</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>130</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">textBrowser-&gt;textBrgb(85, 255, 255);
</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>431</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:36pt;&quot;&gt;北京地铁欢迎您&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>textBrowser</zorder>
   <zorder>startPoint</zorder>
   <zorder>label1</zorder>
   <zorder>label1_2</zorder>
   <zorder>endPoint</zorder>
   <zorder>startConfirm</zorder>
   <zorder>endConfirm</zorder>
   <zorder>startSearch</zorder>
   <zorder>startPointWarn</zorder>
   <zorder>endPointWarn</zorder>
   <zorder>graphicsView</zorder>
   <zorder>administrator</zorder>
   <zorder>textBrowser_2</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SubwayMap</class>
   <extends>QGraphicsView</extends>
   <header>subwaymap.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
